build-aci

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+set -e
+
+if [ $# -ne 1 ]; then
+    echo "Usage: $0 tag" >/dev/stderr
+    exit 1
+fi
+
+tag=$1
+
+tgt=$(mktemp -d)
+
+# Build fleet inside
+docker run -v $PWD:/opt/fleet google/golang:1.3.1 /bin/bash -c "cd /opt/fleet && ./build"
+
+# Generate manifest into target tmp dir
+cat <<DF >${tgt}/manifest
+{
+   "acVersion" : "0.5.1",
```

Comment:
Hmm, I wonder how we should deal with version rot across all these manifests we're embedding.